core-web/libs/sdk/client/src/lib/editor/listeners/listeners.ts

```diff
@@ -100,9 +100,31 @@ export function listenEditorMessages() {
 export function listenHoveredContentlet() {
     const pointerMoveCallback = (event: PointerEvent) => {
         const target = findContentletElement(event.target as HTMLElement);
+
         if (!target) return;
+
         const { x, y, width, height } = target.getBoundingClientRect();
 
+        const isEmptyContainer = target.dataset?.['dotObject'] === 'container';
```

Comment:
This doesn't mean is empty... if there is some kind of padding you can get the target to be the container and actually have content.

Reply:
You need to check the children.

Reply:
I think we already know it is empty at this point, I changed the naming to make it clearer